MonacoEditorComponent/MonacoEditorComponent.nuspec

```
@@ -2,7 +2,7 @@
 <package >
   <metadata>
     <id>Monaco.Editor</id>
-    <version>0.8.1-alpha</version>
+    <version>0.9</version>
```

```suggestion
    <version>0.9-beta</version>
```

We'll still want to have a dash here as it's a "preview" package, though I'll admit that it's getting better, especially with all your great help. 😊